CMakeGraphVizOptions.cmake

```diff
@@ -0,0 +1,23 @@
+# Clean up stuff in OrbitQt
```

Comment:
I guess that file needs a license header

Reply:
done